### ISSUE TITLE
DKIM service isn't ceated by Virtualmin on Ubuntu 18.04

### ISSUE THREAD

**Issue by paulocoghi** [virtualmin/virtualmin-install#25]
Webmin: **1.900**
Virtualmin: **6.06.gpl-2**
OS: **Ubuntu 18.04.2**

As mentioned [here](https://www.virtualmin.com/node/59560) and [here](https://www.virtualmin.com/comment/808804#comment-808804), DKIM service isn't created by Virtualmin on Ubuntu 18.04

The current workaround is to manually create it:

```
/lib/opendkim/opendkim.service.generate
```
```
systemctl daemon-reload
```
```
service opendkim restart
```

And then, the service works. `netstat -natpu | grep opendkim` gives:

```
tcp        0      0 127.0.0.1:8891          0.0.0.0:*               LISTEN      10381/opendkim      
tcp        0      0 127.0.0.1:8891          127.0.0.1:43868         ESTABLISHED 10381/opendkim      
tcp        0      0 127.0.0.1:8891          127.0.0.1:41706         ESTABLISHED 10381/opendkim
```

**Comment by swelljoe** [comment]
This isn't an installer issue. DKIM gets setup later, at least in the current process.

**Comment by paulocoghi** [comment]
Sorry. I forgot to say that this occurs after enabling DKIM on Virtualmin. After enabling it, without the workaround, the outgoing messages don't get signed by opendkim.

**Comment by iliarostovtsev** [comment]
This issue is fixed in Virtualmin 6.10.

**Comment by paulocoghi** [comment]
Thanks for the info, @iliarostovtsev !
